web/src/InstallationFinished.jsx

```diff
@@ -0,0 +1,88 @@
+/*
+ * Copyright (c) [2022] SUSE LLC
+ *
+ * All Rights Reserved.
+ *
+ * This program is free software; you can redistribute it and/or modify it
+ * under the terms of version 2 of the GNU General Public License as published
+ * by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for
+ * more details.
+ *
+ * You should have received a copy of the GNU General Public License along
+ * with this program; if not, contact SUSE LLC.
+ *
+ * To contact SUSE LLC about this file by physical or electronic mail, you may
+ * find current contact information at www.suse.com.
+ */
+
+import React from "react";
+import {
+  Button,
+  Title,
+  Text,
+  EmptyState,
+  EmptyStateIcon,
+  EmptyStateBody,
+  EmptyStateSecondaryActions
+} from "@patternfly/react-core";
+
+import Layout from "./Layout";
+import Center from "./Center";
+import { useInstallerClient } from "./context/installer";
+
+import {
+  EOS_TASK_ALT as InstallationFinishedIcon,
+  EOS_CHECK_CIRCLE as SectionIcon
+} from "eos-icons-react";
+
+const Actions = ({ onRestart, onFinish }) => (
+  <>
+    <Button isLarge variant="secondary" onClick={onRestart}>
+      Restart Installation
+    </Button>
+    <Button isLarge variant="primary" onClick={onFinish}>
+      Finish
```

Comment:
By now, we are not implementing a Reboot action. What about just a "Close" button? It could close the tab.

Reply:
Good idea, but sadly `window.close()` does not work because

> Scripts may not close windows that were not opened by script.

I had try some dirty tricks, but none of them worked. So, I removed the button and adapted the feedback test. See the PR screenshot.